### DISCUSSION TITLE
spinners and sliders should have different "Icons" on the editor timeline

### DISCUSSION THREAD

**Discussion by Stoppedpuma** [ppy/osu#18452] [Ideas]
Currently both spinners and sliders have the same icon on the editor timeline and should probably be different from each other.

Current:
![osu!_423xVzavGP](https://user-images.githubusercontent.com/58333920/170813387-15f130ec-9527-4727-bab5-015b9c40757e.png)
 Idea:
![spinneridea](https://user-images.githubusercontent.com/58333920/170813404-92b72ab0-4c00-4feb-83ec-4787c664e6cd.png)

**Comment by peppy** [comment]
Right now, the timeline is ruleset agnostic and we're avoiding doing anything like this. Might be something we introduce in a distant future.
